ui/lib/core/addon/templates/components/form-error.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="is-flex is-flex-center">
```

Comment:
maybe check with Ivana if there should be any padding/margin on this component.  Seems a little close to the text. 
![image](https://user-images.githubusercontent.com/6618863/95113215-5789ae00-06ff-11eb-8ed4-e831fa84c990.png)